zerver/worker/queue_processors.py

```diff
@@ -427,6 +430,18 @@ def consume(self, event):
 class OutgoingWebhookWorker(QueueProcessingWorker):
     def consume(self, event):
         # type: (Mapping[str, Any]) -> None
-
-        # TODO: Implement this worker.
-        pass
+        message = event['message']
+        services = get_bot_services(event['user_profile_id'])
+        rest_operation = {'method': 'POST',
+                          'relative_url_path': '',
+                          'request_kwargs': {},
+                          'base_url': ''}
+        dup_event = dict()
```

Comment:
Here I want to pass event to following functions so that it can be used during retries i.e. the same object can be inserted into the queue again in case of retry, with just the retry count incremented. But I was getting mypy errors on editing both original and copied event structure. Hence I initialised a new dict() using brute force i.e. accessing key and value pairs from event and storing them. 
I think both shallow and deep copy would work for me but I am getting mypy errors on editing both of them.